simple_config

```diff
@@ -1,5 +1,6 @@
 # This is a comment
 port 2020;
+num_threads 5;
 
```

Comment:
There should definitely be a default value for number of threads even if a user does not specify it in the config file (like 6 or something).